_includes/integrations/vue-tech-ref.md

```diff
@@ -0,0 +1,367 @@
+- [Configuring the editor](#configuringtheeditor)
+
+    - [`api-key`](#api-key)
+    - [`cloud-channel`](#cloud-channel)
+    - [`disabled`](#disabled)
+    - [`id`](#id)
+    - [`init`](#init)
+    - [`initial-value`](#initial-value)
+    - [`inline`](#inline)
+    - [`modelEvents`](#modelevents)
+    - [`plugins`](#plugins)
+    - [`tag-name`](#tag-name)
+    - [`toolbar`](#toolbar)
+
+- [Form Input Bindings: `v-model`](#forminputbindingsv-model)
+- [Event binding](#eventbinding)
+
+### Installing the TinyMCE Vue.js integration using NPM
+
+To install the `tinymce-vue` package and save it to your `package.json`.
+
+    ```
+    $ npm install --save @tinymce/tinymce-vue
+    ```
+
+### Using the TinyMCE Vue.js integration
+
+1. Load the component.
+    * For bundle loader users (such as `webpack`, `rollup`, or `browserify`):
+
+        ```js
+        // es modules
+        import Editor from '@tinymce/tinymce-vue';
+        // commonjs require
+        // NOTE: default needed after require
+        var Editor = require('@tinymce/tinymce-vue').default;
+        ```
+    * To load `tinymce-vue` in a javascript file:
+
+        ```html
+        <script src="/path/to/tinymce-vue.min.js"></script>
+        ```
+2. Add the editor to the `components` property of the app:
+
+    ```js
+    // This might look different depending on how you have set up your app
+    // but the important part is the components property
+    var app = new Vue({
+      el: '#app',
+      data: { /* Your data */ },
+      components: {
+        'editor': Editor // <- Important part
+      },
+      methods: { /* Your methods */}
+    })
+    ```
+3. Add the `<editor>` tag to the `template`
+
+    ```html
+    <editor api-key="API_KEY" :init="{plugins: 'wordcount'}"></editor>
+    ```
+
+### Configuring the editor
+
+The editor accepts the following properties:
+
+```xml
+<editor
+  api-key="your-api-key"
+  cloud-channel="5"
+  :disabled=false
+  id="uuid"
+  :init= "{% raw %}{{% endraw %}  {% raw %}}{% endraw %}"
+  initial-value=""
+  :inline=true
+  modelEvents= ""
+  plugins=""
+  tag-name="div"
+  toolbar=""
+  value=""
+/>
+```
+
+None of the configuration properties are **required** for `tinymce-vue` to work. Specify a Tiny Cloud API key using `api-key` to remove the `This domain is not registered...` warning message.
+
+#### `api-key`
+Tiny API key. Required for deployments using the Tiny Cloud to provide the TinyMCE editor.
+
+To register for a Tiny Cloud API key, visit the [sign-up page](https://www.tiny.cloud/signup/).
+
+**Default value:** `no-api-key`
+
+**Type:** String
+
+##### Example: `api-key`
+
+```xml
+<editor
+  api-key="your-api-key"
+/>
+```
+
+#### `cloud-channel`
+
+**Default value:** `5`
+
+**Possible values:**  `5-stable`, `5-testing`, `5-dev`
+
+Changes the TinyMCE build used for the editor to one of the following cloud channels:
+
+- `5`, `5-stable` (**Default**): The current enterprise release of TinyMCE.
```

Comment:
Do we need default mentioned here again, given we have it above?

Reply:
I'd like to leave it that way to reduce confusion